index.html

```diff
@@ -1,14 +1,164 @@
+          <button class="button2" type="button"><a target="_blank" href="continue-reading.html">CONTINUE READING</a></button>
+          <br>
+          <p class="underline">                                                            </p>
+          <h4 class="main-content1-text1">AWESOME DEMO PHOTO</h4>
+          <img class="main-content1-img1" src="https://picsum.photos/300/203"/>
+          <p class="img1-desc1">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Eos in sint, minus qui eius aliquid id molestiae sunt</p>
+          <p>reiciendis atque? Cupiditate nam minima provident aut fugit tempora recusandae ab, dolor unde labore soluta sint reiciendis velit, eum molestias aspernatur praesentium, fuga dolorem non fugiat perferendis nesciunt. Porro nesciunt ex cum unde exercitationem corporis expedita deleniti esse? Explicabo a amet eligendi at non molestias facilis autem doloremque. Quos dolore placeat illum? Sunt culpa perferendis quas distinctio</p> 
+          <p>fugit veniam tenetur est illum corrupti, ipsum accusamus doloremque laborum accusantium aut hic consequuntur delectus! Officiis repellat distinctio voluptas nostrum unde eveniet cupiditate non eligendi.</p>
+          <button class="button2" type="button"><a target="_blank" href="continue-reading.html">CONTINUE READING</a></button>
+          <br>
+          <br>
+          <br>
+          <br>
+          <br>
+        </div>
+      
+        <div class="main-content1-sidebar">
+          <br>
+          <div class="date-container">
+            <div class="date">JULY 30</div>
+            <h5 class="sidebar-post-title">JUST ANOTHER POST</h5>
+            <p class="sidebar-post">Lorem ipsum dolor sit amet consectetur adipisicing elit. Officiis atque magnam minima, corporis quibusdam et doloribus ipsum ad cum expedita!</p>
+            <br>
+          </div>
+          <br>
+          <br>
+          <div class="date-container">
+            <div class="date">JULY 28</div>
+            <h5 class="sidebar-post-title">AND ANOTHER POST</h5>
+            <p class="sidebar-post">Lorem ipsum dolor sit amet consectetur adipisicing elit. Officiis atque magnam minima, corporis quibusdam et doloribus ipsum ad cum expedita!</p>
+            <br>
+          </div>
+          <br>
+          <div class="date-container2">
+            <div class="date">JULY 24</div>
+            <h5 class="sidebar-post-title">ONE MORE POST</h5>
+            <p class="sidebar-post">Lorem ipsum dolor sit amet consectetur adipisicing elit. Officiis atque magnam minima, corporis quibusdam et doloribus ipsum ad cum expedita!</p>
+            <br>
+          </div>

```

Comment:
These should have links connected to your "Blog-Post.HTML" page

Reply:
I didn't see any link buttons in the example so I don't put any in. But I can add the in. Maybe have it link with the "date" text?